workflows/tpv33/main.workflow

```diff
@@ -0,0 +1,31 @@
+workflow "containers" {
+  on = "push"
+  resolves = "execute"
+}
+
+action "build and test" {
+  uses = "./actions/seissol"
+  args = [
+   "compileMode=release",
+   "order=6",
+   "parallelization=hybrid",
+   "netcdf=yes",
+   "hdf5=yes",
+   "commThread=yes",
+   "compiler=gcc",
+   "unitTests=fast",
+   "-j8",
+  ]
+}
```

Comment:
and add another action named `test` that runs unit tests (same as above, but appending `check` to the args list)